c++/src/Statistics.cc

```diff
@@ -327,6 +329,10 @@ namespace orc {
                 (stats.has_maximum() && (statContext.writerTimezone != nullptr)));
       _hasLowerBound = stats.has_minimumutc() || stats.has_minimum();
       _hasUpperBound = stats.has_maximumutc() || stats.has_maximum();
+      _minimumNano = stats.has_minimumnanos() ?
+                     stats.minimumnanos() - 1 : DEFAULT_MIN_NANOS;
```

Comment:
Could you add some comment about why why we use `-1` here?

Reply:
sure